entities/Search.ts

```diff
@@ -112,6 +112,14 @@ export class Search {
         return response as Promise<PixivAutoCompleteV2>
     }
 
+    /**
+     * Gets suggested candidates from web url that includes tag translation and access count.
+     */
+    public searchForCandidates = async (params?: PixivWebParams) => {
```

Comment:
Name is a bit lengthy I prefer if it's just called `candidates`. Also I don't know if putting this in `search` is right because it doesn't return `next_url` like the other functions. I would probably create a new class and put this under `pixiv.web.candidates()`.

Reply:
I don't think a new class is needed? It returns almost the same value as `autoCompleteV2`. As for now, I will leave it in `search`.